packages/aws-cdk-lib/aws-opensearchservice/lib/version.ts

```diff
@@ -99,6 +99,12 @@ export class EngineVersion {
   /** AWS OpenSearch 2.11 */
   public static readonly OPENSEARCH_2_11 = EngineVersion.openSearch('2.11');
 
+  /** AWS OpenSearch 2.12 */
+  public static readonly OPENSEARCH_2_12 = EngineVersion.openSearch('2.12');
```

Comment:
I don't see 2.12 included in the [supported versions](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html#choosing-version).

Reply:
my bad, I just assumed it was continuous.

Removed 2.12 now